specification/signalr/resource-manager/Microsoft.SignalRService/preview/2020-07-01-preview/scenarios/peArmTemplate/template.json

```diff
@@ -0,0 +1,200 @@
+{
```

Comment:
Isn't this file deleted in the last commit?

Reply:
seems not, just removed it, and also remove it from the GitHub PR